schemas/action-schema.json

```diff
@@ -56,6 +56,27 @@
                   "type": "string",
                   "description": "Target name",
                   "readOnly": false
+                },
+                "classname": {
+                  "type": "string",
+                  "description": "Target class name",
+                  "readOnly": false
+                },
+                "parent": {
+                  "type": "object",
+                  "description": "Parent element of the target",
+                  "properties": {
+                    "classname": {
+                      "type": "string",
+                      "description": "Class name of the parent element",
+                      "readOnly": false
+                    }
+                  }
+                },
+                "resource_id": {
+                  "type": "string",
+                  "description": "identifier for the target resource",
+                  "readOnly": true
                 }
```

Comment:
Is it a RUM resource event id? Maybe it should be `resource.id` like other id fields.

Reply:
I think the fields are already sent by the SDK, @xgouchet might have more insight on this ?

Reply:
No those are Android resource ids, nothing to do with RUM Resources. cf https://developer.android.com/guide/topics/resources/providing-resources

Reply:
Then IMO it should be a bit more explicit, like `android_resource_id`